chainer/functions/connection/n_step_rnn.py

```diff
@@ -0,0 +1,889 @@
+            Each ``ws[i + di]`` is a list containing two matrices.
+            ``ws[i + di][j]`` is corresponding with ``W^{f}_j`` if ``di = 0``
+            and corresponding with ``W^{b}_j`` if ``di = 1`` in the equation.
+            Only ``ws[0][j]`` and ``ws[1][j]`` where ``0 <= j < 1`` are
+            ``(I, N)`` shape as they are multiplied with input variables.
+            All other matrices has ``(N, N)`` shape.
+        bs (list of list of chainer.Variable): Bias vectors. ``bs[i + di]``
+            represnents biases for i-th layer.
+            Note that ``di = 0`` for forward-RNN and ``di = 1`` for
+            backward-RNN.
+            Each ``bs[i + di]`` is a list containing two vectors.
+            ``bs[i + di][j]`` is corresponding with ``b^{f}_j`` if ``di = 0``
+            and corresponding with ``b^{b}_j`` if ``di = 1`` in the equation.
+            Shape of each matrix is ``(N,)`` where ``N`` is dimention of
+            hidden units.
+        xs (list of chainer.Variable): A list of :class:`~chainer.Variable`
+            holding input values. Each element ``xs[t]`` holds input value
+            for time ``t``. Its shape is ``(B_t, I)``, where ``B_t`` is
+            mini-batch size for time ``t``, and ``I`` is size of input units.
+            Note that this functions supports variable length sequences.
+            When sequneces has different lengths, sort sequences in descending
+            order by length, and transpose the sorted sequence.
+            :func:`~chainer.functions.transpose_sequence` transpose a list
+            of :func:`~chainer.Variable` holding sequence.
+            So ``xs`` needs to satisfy
+            ``xs[t].shape[0] >= xs[t + 1].shape[0]``.
+        train (bool): If ``True``, this function executes dropout.
+        use_cudnn (bool): If ``True``, this function uses cuDNN if available.
+        activation (str): Activation function name.
+            Please select ``tanh`` or ``relu``.
+
+    Returns:
+        tuple: This functions returns a tuple concaining three elements,
+            ``hy`` and ``ys``.
+
+            - ``hy`` is an updated hidden states whose shape is same as ``hx``.
+            - ``ys`` is a list of :class:`~chainer.Variable` . Each element
+              ``ys[t]`` holds hidden states of the last layer corresponding
+              to an input ``xs[t]``. Its shape is ``(B_t, N)`` where ``B_t``
+              is mini-batch size for time ``t``, and ``N`` is size of hidden
+              units. Note that ``B_t`` is the same value as ``xs[t]``.
+
+    """
+    return n_step_rnn_base(n_layers, dropout_ratio, hx, ws, bs, xs, train,
+                           use_cudnn, activation, use_bi_direction=True)
+
+
+def n_step_rnn_base(n_layers, dropout_ratio, hx, ws, bs, xs, train,
+                    use_cudnn, activation, use_bi_direction):
+    """Stacked RNN Base function for sequence inputs.
+
+    This function calculates stacked RNN with sequences. This function gets
```

Comment:
This description is incorrect. Its behavior depends on `activation` and `use_bi_direction`. So all you need to write here is that this function is the base function of all these functions, its behavior depends on arguments and links to the concrete methods.